go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -83,11 +85,11 @@ func init() {
 func registerThrottlerFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&throttleTabletTypes, "throttle_tablet_types", throttleTabletTypes, "Comma separated VTTablet types to be considered by the throttler. default: 'replica'. example: 'replica,rdonly'. 'replica' aways implicitly included")
 
-	fs.DurationVar(&throttleThreshold, "throttle_threshold", throttleThreshold, "Replication lag threshold for default lag throttling")
-	fs.StringVar(&throttleMetricQuery, "throttle_metrics_query", throttleMetricQuery, "Override default heartbeat/lag metric. Use either `SELECT` (must return single row, single value) or `SHOW GLOBAL ... LIKE ...` queries. Set -throttle_metrics_threshold respectively.")
-	fs.Float64Var(&throttleMetricThreshold, "throttle_metrics_threshold", throttleMetricThreshold, "Override default throttle threshold, respective to --throttle_metrics_query")
-	fs.BoolVar(&throttlerCheckAsCheckSelf, "throttle_check_as_check_self", throttlerCheckAsCheckSelf, "Should throttler/check return a throttler/check-self result (changes throttler behavior for writes)")
-	fs.BoolVar(&throttlerConfigViaTopo, "throttler-config-via-topo", throttlerConfigViaTopo, "When 'true', read config from topo service and ignore throttle_threshold, throttle_metrics_threshold, throttle_metrics_query, throttle_check_as_check_self")
+	fs.DurationVar(&deprecatedThrottleThreshold, "throttle_threshold", deprecatedThrottleThreshold, "Replication lag threshold for default lag throttling")
+	fs.StringVar(&deprecatedThrottleMetricQuery, "throttle_metrics_query", deprecatedThrottleMetricQuery, "Override default heartbeat/lag metric. Use either `SELECT` (must return single row, single value) or `SHOW GLOBAL ... LIKE ...` queries. Set -throttle_metrics_threshold respectively.")
+	fs.Float64Var(&deprecatedThrottleMetricThreshold, "throttle_metrics_threshold", deprecatedThrottleMetricThreshold, "Override default throttle threshold, respective to --throttle_metrics_query")
+	fs.BoolVar(&deprecatedThrottlerCheckAsCheckSelf, "throttle_check_as_check_self", deprecatedThrottlerCheckAsCheckSelf, "Should throttler/check return a throttler/check-self result (changes throttler behavior for writes)")
+	fs.BoolVar(&deprecatedThrottlerConfigViaTopo, "throttler-config-via-topo", deprecatedThrottlerConfigViaTopo, "Deprecated, will be removed in v19. Assumed to be 'true'")
```

Comment:
Any reason we're not leveraging pFlag's deprecated flag feature?

Reply:
done